src/js/BookReader/Mode2Up.js

```diff
@@ -664,6 +665,10 @@ export class Mode2Up {
           this.br.animationFinishedCallback();
           this.br.animationFinishedCallback = null;
         }
+
+        this.br.refs.$brContainer.removeClass("BRpageFlipping");
+
+        if(this.br.enableTextSelection) this.br.textSelectionPlugin.stopPageFlip(this.br.refs.$brContainer);
```

Comment:
This is still a little awkward; is there any way we can avoid having this function exist?

Reply:
Tried to add an event to event.js, but I was not able to pass the $brContainer as a parameter

Reply:
What is $brContainer? Can `TextSelectionPlugin` take a copy of that element and hold on to it? Or does it change?

Reply:
Actually nevermind; it's ok. I don't think that's worth implementing right now. But if there are other options :)